components/bunnygrabber.py

```diff
@@ -1,19 +1,31 @@
+# import Robot py library so this file can interface with the robot.
 import wpilib
 
-
+# Class for controlling arm, will be instantiated in later script usage.
 class BunnyGrabber:
-    """Controls the bunny grabber.
-    """
+#	This program controls the bunny grabber arm.
```

Comment:
We should be using Python [docstrings](https://www.python.org/dev/peps/pep-0257/) for documenting classes and functions, not comments